cmd/generate/config/rules/config.tmpl

```diff
@@ -29,6 +29,8 @@ paths = [
 
 {{ range $i, $rule := .Rules }}[[rules]]
 id = "{{$rule.RuleID}}"
+{{- with $rule.Enabled }}
```

Comment:
This probably doesn't work as intended. I think we'd want it the other way: include if `false`, exclude if `true`.